db/migrate/20240405075558_remove_pacbio_pool_id_from_pacbio_library.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+# Remove the Pacbio libraries with a pacbio_pool_id 
+# and remove the pacbio_pool_id column from the pacbio_libraries table
+class RemovePacbioPoolIdFromPacbioLibrary < ActiveRecord::Migration[7.1]
+  def up
+    # Remove all libraries that have a pool_id as they have been replaced by aliquots
+    Pacbio::Library.transaction do
+      Pacbio::Library.where.not(pacbio_pool_id: nil).each do |library|
+        library.delete
+      end
+    end
+
+    # Remove the column from the table
+    remove_reference :pacbio_libraries, :pacbio_pool, null: true, foreign_key: true
+  end
+
+  def down
+    raise ActiveRecord::IrreversibleMigration
```

Comment:
I do understand why this is irreversible because of the data loss that would occur when moving `up`.  Would it be better to not make this irreversible and just arrange the tables back the other way again even if the data is missing, or is that not possible because there must be data in those columns?